mteb/models/overview.py

```diff
@@ -11,7 +11,8 @@
 from mteb.abstasks.AbsTask import AbsTask
 from mteb.encoder_interface import Encoder
 from mteb.model_meta import ModelMeta
-from mteb.models import (
+
+from . import (
```

Comment:
I tried to resolve the circular error here, worked locally but didn't quite work in CI. Feel free to revert during your fix.